tools/sense_studio/video_recording.py

```diff
@@ -14,6 +14,12 @@
 video_recording_bp = Blueprint('video_recording_bp', __name__)
 
 
+@video_recording_bp.route('/ffmpeg-check')
+def check_ffmpeg():
+    ffmpeg_installed = os.popen("dpkg-query -W -f='${Status}' ffmpeg").read()
```

Comment:
I don't think we can assume that everyone is using `dpkg` (e.g. on a different OS). Can you maybe try running `ffmpeg -version` instead?

Reply:
> I don't think we can assume that everyone is using `dpkg` (e.g. on a different OS). Can you maybe try running `ffmpeg -version` instead?

update